src/cascadia/TerminalApp/TerminalSettings.cpp

```diff
@@ -12,6 +12,25 @@ using namespace winrt::Microsoft::Terminal::Settings::Model;
 
 namespace winrt::TerminalApp::implementation
 {
+    static constexpr std::array<til::color, 16> campbellColorTable{

```

Comment:
Why can't we use the one from Utils? Is it because it's a filler-style method?

Maybe we can just expose a function that returns it directly... as a gsl::span.

Reply:
(Really don't love the duplication)

Reply:
Yeah I don't like the duplication either - will add a function to `colorTable.hpp` to expose the campbell color table